exec/cnex/global.c

```diff
@@ -1218,6 +1218,46 @@ void object__invokeMethod(TExecutor *exec)
         goto cleanup;
     }
 
+    if (obj->object->type == oArray) {
+        if (strcmp(name, "size") == 0) {
+            if (args->array != NULL && args->array->size != 0) {
+                exec->rtl_raise(exec, "DynamicConversionException", "invalid number of arguments to size() (expected 0)");
+                goto cleanup;
+            }
+            push(exec->stack, cell_fromObject(object_createNumberObject(number_from_uint64(((Cell*)obj->object->ptr)->array->size))));
+            goto cleanup;
+        }
```

Comment:
Need an `else` clause here to catch unknown method names and raise an exception.

Reply:
Fixed.